neo/Settings.py

```diff
@@ -203,6 +205,9 @@ def setup(self, config_file):
         if 'ServiceEnabled' in config:
             self.SERVICE_ENABLED = bool(config['ServiceEnabled'])
 
+        if 'CompilerNep8' in config:
+            self.COMPILER_NEP_8 = bool(config['CompilerNep8'])
```

Comment:
I'd like to suggest to at least use `config.get('CompilerNep8', False)` because the `AcceptIncomingPeers` update showed that quite some users just copy/replace their configs with their old one after they upgraded. Then this call will fail because the key is not there. Ideally we might want to have a utility function that tries to get the key, if not uses a specified default value but also warns the user that the key was not found and is now using the "x" as the default.